app/uk/gov/hmrc/cataloguefrontend/service/PlatformInitiativesService.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cataloguefrontend.service
+
+import laika.api.Transformer
+import laika.format.Markdown
+import laika.markdown.github.GitHubFlavor
+
+class PlatformInitiativesService {
```

Comment:
The name doesn't seem to match the responsibility. Maybe rename to `MarkdownUtils`? It could also be an object, rather than a class - which will make it easier to use/test.
Have you reviewed other places we use Markdown in catalogue? There may be some duplication we could extract to a common utility?

Reply:
Will add the markdown method to the `MarkdownLoader` utils file as per our discussion earlier